doc/api/util.md

```diff
@@ -369,9 +369,23 @@ Returns the string name for a numeric error code that comes from a Node.js API.
 The mapping between error codes and error names is platform-dependent.
 See [Common System Errors][] for the names of common errors.
 
+## `util.getSystemErrorMap()`
+<!-- YAML
+added: v9.7.0
+-->
+
+* Returns: {Map}
+
+Returns a Map of all system error codes available from Node.js API.
+The mapping between error codes and error names is platform-dependent.
+See [Common System Errors][] for the names of common errors.
+
+
+
 ```js
 fs.access('file/that/does/not/exist', (err) => {
-  const name = util.getSystemErrorName(err.errno);
+  const errorMap = util.getSystemErrorMap();
+  const name = errorMap.get(err.errno);
```

Comment:
The example looks good for the new `util.getSystemErrorMap()` but we should probably also keep the existing example for `util.getSystemErrorName(err)`.

Reply:
that was accidental :( fixing it